samples/grpc/iot-service/README.md

```diff
@@ -0,0 +1,44 @@
+## Running the sample code
+
+1. Start a local PostgresSQL server on default port 5432. The included `docker-compose.yml` starts everything required for running locally.
+
+    ```shell
+    docker compose up --wait
+
+    # creates the tables needed for Akka Persistence
+    # as well as the offset store table for Akka Projection
+    docker exec -i postgres_db psql -U postgres -t < ddl-scripts/create_tables.sql
+    ```
+
+2. Start a first node:
+
+    ```shell
+    sbt -Dconfig.resource=local1.conf run
+    ```
+
+3. (Optional) Start another node with different ports:
+
+    ```shell
+    sbt -Dconfig.resource=local2.conf run
+    ```
+
+4. (Optional) More can be started:
+
+    ```shell
+    sbt -Dconfig.resource=local3.conf run
+    ```
+
+5. Check for service readiness
+
+    ```shell
+    curl http://localhost:9101/ready
+    ```
+
+6. Try it with [grpcurl](https://github.com/fullstorydev/grpcurl).
+
+    ```shell
+    # register
+    grpcurl -d '{"sensor_id":"temp-1", "secret":"foo"}' -plaintext 127.0.0.1:8101 iot.registration.RegistrationService.Register
```

Comment:
@johanandren something is wrong, and I can't see what would be different compared to the shopping-cart-service-scala

This returns:
```
grpcurl -d '{"ssensor_id":"temp-1", "secret":"foo"}' -plaintext 127.0.0.1:8101 iot.registration.RegistrationService.Register
Error invoking method "iot.registration.RegistrationService.Register": target server does not expose service "iot.registration.RegistrationService"
```

it is listed:
```
grpcurl -plaintext 127.0.0.1:8101 list
grpc.reflection.v1alpha.ServerReflection
iot.registration.RegistrationService
```

Reply:
I think this may be the problem: https://github.com/akka/akka-projection/pull/955/files#diff-14e222f54a5cd0429a823860b653acf84d1ec041af1e5413f746e2d1b4fa1142R62

The push handler isn't actually partial, it's complete (but I would have thought that would block accessing any other services including the reflection service, so I may be wrong)

Reply:
Ok, and that is always partial, so that's not the cause, hmmm 🤔

Reply:
Something weird with your machine? I checked it out and it works when I run the current PR state locally.

Reply:
strange, but good, thanks